src/tools/tidy/src/pal.rs

```diff
@@ -62,6 +62,8 @@ const EXCEPTION_PATHS: &[&str] = &[
     "library/std/src/panic.rs",   // fuchsia-specific panic backtrace handling
     "library/std/src/personality.rs",
     "library/std/src/personality/",
+    "library/std/src/thread/local.rs", // UEFI special thread_local implementation
+    "library/std/src/thread/mod.rs",   // UEFI specific thread_local implementation
```

Comment:
Well, not adding them causes failure in running tidy checks with the following output:
```sh
tidy error: /var/home/ayush/Documents/Programming/Rust/rust/uefi-std-minimal/library/std/src/thread/local.rs:199: platform-specific cfg: cfg(any(all(target_family = "wasm", not(target_feature = "atomics")), target_os = "uefi"))
tidy error: /var/home/ayush/Documents/Programming/Rust/rust/uefi-std-minimal/library/std/src/thread/local.rs:206: platform-specific cfg: cfg(all(
                target_thread_local,
                not(target_os = "uefi"),
                not(all(target_family = "wasm", not(target_feature = "atomics"))),
            ))
tidy error: /var/home/ayush/Documents/Programming/Rust/rust/uefi-std-minimal/library/std/src/thread/local.rs:263: platform-specific cfg: cfg(all(
                not(target_thread_local),
                not(target_os = "uefi"),
                not(all(target_family = "wasm", not(target_feature = "atomics"))),
            ))
tidy error: /var/home/ayush/Documents/Programming/Rust/rust/uefi-std-minimal/library/std/src/thread/local.rs:326: platform-specific cfg: cfg(any(all(target_family = "wasm", not(target_feature = "atomics")), target_os = "uefi"))
tidy error: /var/home/ayush/Documents/Programming/Rust/rust/uefi-std-minimal/library/std/src/thread/local.rs:331: platform-specific cfg: cfg(all(
                    target_thread_local,
                    not(target_os = "uefi"),
                    not(all(target_family = "wasm", not(target_feature = "atomics"))),
                ))
tidy error: /var/home/ayush/Documents/Programming/Rust/rust/uefi-std-minimal/library/std/src/thread/local.rs:339: platform-specific cfg: cfg(all(
                    not(target_thread_local),
                    not(target_os = "uefi"),
                    not(all(target_family = "wasm", not(target_feature = "atomics"))),
                ))
tidy error: /var/home/ayush/Documents/Programming/Rust/rust/uefi-std-minimal/library/std/src/thread/local.rs:870: platform-specific cfg: cfg(any(all(target_family = "wasm", not(target_feature = "atomics")), target_os = "uefi"))
tidy error: /var/home/ayush/Documents/Programming/Rust/rust/uefi-std-minimal/library/std/src/thread/mod.rs:241: platform-specific cfg: cfg(any(all(target_family = "wasm", not(target_feature = "atomics")), target_os = "uefi"))
```

This means if the git hooks are installed, it will not allow you to commit.